guides/doc-Provisioning_Guide/topics/proc_installing_ipv6_environment.adoc

```diff
@@ -0,0 +1,203 @@
+= Installing IPv6 environment
+
+{Project} allows provisioning of {RHEL} 8 or newer over IPv6 provisioning networks. Although provisioning templates which ship with {Project} include IPv6 support for PXE and HTTP (iPXE) scenarios, the only tested and certified workflow is UEFI HTTP Boot with unmanaged DHCPv6 server which is described in this chapter.
+
+== Limitations
+
+DHCP server in {RHEL} (ISC DHCP) does not provide an integration API for managing IPv6 records, therefore {SmartProxy} DHCP plugin that would provide DHCP management of IPv6 subnets does not exist. In this scenario, DHCP IPv6 must be deployed as a separate (unmanaged) service to boostrap client nodes into grub2 which then configures IPv6 networking either using DHCPv6 or via static IPv6 address.
+
+== Requirements
+
+ifeval::["{build}" == "satellite"]
+{SmartProxy} must be installed on {RHEL} version 7.9 or higher.
++
+{Project} or {SmartProxy} can be only installed on IPv6-only systems, dualstack is known to work but not supported.
```

Comment:
Here you use `dualstack`, later in the document you use `dual-stack`. Not sure which one is right, but we should use only one of the forms